tsconfig.json

```diff
@@ -58,5 +58,5 @@
     // "emitDecoratorMetadata": true,         /* Enables experimental support for emitting type metadata for decorators. */
     // "maxNodeModuleJsDepth": 5
   },
-  "include": ["./src/y-websocket.js"]
+  "include": ["./src/y-websocket.js", "./bin/utils.js"]
```

Comment:
Right, that makes total sense!

But is it possible to keep `outDir: "./dist"`? Or did it lead to issues?

Reply:
I will look into that! (it must have caused some issue or else why did I make so many changes lol. Lemme see if I can figure out what the issue was)